CHANGELOG.md

```diff
@@ -4,6 +4,124 @@
 
 <!-- next version -->
 
+## v0.24.0
+
+This release includes version 0.120.0/0.120.1 of the upstream Collector components.
+
+The individual upstream Collector changelogs can be found here:
+
+v0.120.1:
+
+- <https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.120.1>
+
+v0.120.0:
+
+- <https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.120.0>
+- <https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.120.0>
+
+<details>
+<summary>Highlights from the upstream Collector changelog</summary>
+
+### 🛑 Breaking changes 🛑
```

Comment:
I wonder if we should put the breaking changes above the collapsible section.
Otherwise, it's very very subtle and easy to miss, specifically the highly relevant prometheusreceiver one this time.

![image](https://github.com/user-attachments/assets/790e9ffd-8012-4f21-9e52-df3e3a3d22f7)

Reply:
good suggestion, that definitely makes sense to do

Reply:
Thanks! Could you perhaps apply the change also to the last few releases (maybe 0.20.0+?) so it's consistent?
They have only 2-5 entries each luckily so it's not too cluttering.

Reply:
done :)

Reply:
One last nit 😄 I think a h4 would be enough for the breaking changes section such that the h2 version number is still the clear separator - but the markdown linter might complain if #### follows ## directly.

Reply:
seems like the linter allows it 👍